include/sdf/SDFImpl.hh

```diff
@@ -64,6 +65,21 @@ namespace sdf
                        bool _searchLocalPath = true,
                        bool _useCallback = false);
 
+  /// \brief Find the absolute path of a file.
```

Comment:
This new function, and the old flavor, don't fully specify how the searching actually takes place. (I understand it's implementation specific, but still.)

From `_useCallback`, I infer that "normal mechanism" means `_searchLocalPath`, but I don't know it for sure.
Can you make this text more concise, and tell what the priorities are?

Reply:
I don't know about more concise, but I listed the search order in https://github.com/osrf/sdformat/commit/6f53a9f113b1d2b8e4749ee5a55f76913b9f162b. How does that look?

Reply:
Ooh, that's excellent! Thank you!